jekyll/_cci2/testing-macos.md

```diff
@@ -33,6 +31,12 @@ While it is possible to write to the copy that is located in the home directory,
 
 While adding permissions can be manually written in your CircleCI config with `sqlite3` commands, [CircleCI provides an Orb](https://circleci.com/developer/orbs/orb/circleci/macos) to simplify this.
 
+## Supported Xcode and macOS Versions
+
+Testing macOS apps is only supported on Xcode 11.7 images and newer.
```

Comment:
Maybe it's worth putting something about why? Because SIP is disabled for these images? Or maybe it's more complex than that?